crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -635,6 +635,10 @@ define_tedge_config! {
             },
         },
 
+        /// The entity which should start tedge File Transfer Service
+        #[tedge_config(default(value = "device/main//"), example = "device/main//", example = "device/child//")]
+        entity: Arc<str>,
```

Comment:
No, there is a small difference. This setting is not "what is the entity id of the current device", but "the agent of which device should spawn the FTS?"

First, we don't want to only enable the FTS on the main device, because usually the c8y mapper runs on the main device, so the FTS should run on child devices as well. So I first tried to create `enable.file_transfer_service`, to be set to true on a device when we want to start it, and false when we don't.
But what should be the default value?
1. If it's `false` then by default FTS won't be started on a main device, and we'll introduce a breaking change where it would be necessary to set this config to `true` to enable it back on.
2. If the default is `true`, then it will start on a main device, but also on any child device as well, so we'd be forced to always manually set it to `false` on any child device.
3. If we used a combination of "start if running on a main device, and if not, then use the `enable.file_transfer_service` with default of `false`" then we wouldn't be able to disable the FTS for the main device.

Using an explicit entity topic id solves that problem. By default, it's the main device, so the behaviour is unchanged and everything works. If it's changed to a child device, then only that child device agent starts the FTS and other agents don't.
One problem is that we still need to change configuration not in 1, but 2 places for it to work: the child device on which we plan to run FTS on, and on the main device so it doesn't run there by default. For the rest of the child devices, the default value of `device/main//` will prevent their agents from starting, so we don't have to change anything on them. Since [the plan is to have abstraction over a configuration using MQTT or HTTP API so that all devices don't need to have access to the same filesystem to change or read the configuration](https://github.com/thin-edge/thin-edge.io-roadmap/issues/44), perhaps we could then only set it in 1 place.